media/videostitcher/create_cdn_key.go

```diff
@@ -25,16 +25,16 @@ import (
 )
 
 // createCdnKey creates a CDN key. A CDN key is used to retrieve protected media.
-// If isMediaCdn is true, create a Media CDN key. If false, create a Cloud
+// If isMediaCDN is true, create a Media CDN key. If false, create a Cloud
 // CDN key. To create a privateKey value for Media CDN, see
 // https://cloud.google.com/video-stitcher/docs/how-to/managing-cdn-keys#create-private-key-media-cdn.
-func createCdnKey(w io.Writer, projectID, cdnKeyID, hostname, keyName, privateKey string, isMediaCdn bool) error {
+func createCdnKey(w io.Writer, projectID, keyID, hostname, keyName, privateKey string, isMediaCDN bool) error {
```

Comment:
Function names should also use `CDN` over `Cdn`.

Reply:
Done